packages/design-system/src/styles/components/_ChoiceList.scss

```diff
@@ -9,46 +9,44 @@
 
 // Layout styles for wrapping choice container
 .ds-c-choice-wrapper {
-  display: flex;
-
-  // Medicare has a different gap for choice wrappers
-  // If a theme doesn't have a gap for choice wrappers defined, it will fall back to the normal gap
-  gap: var(--choice-wrapper__gap, 0.5rem);
+  column-gap: var(--choice-wrapper__gap, 0.5rem);
+  display: grid;
+  grid-template-areas:
+    'check label'
+    'check hint'
+    'check error-message';
+  grid-template-columns: min-content 1fr;
   margin-block-start: $spacer-1;
   max-width: var(--measure-wide);
+  row-gap: 0;
```

Comment:
i don't think this line's needed

Reply:
You're right. This was left-over from when I had the wrapper class on the root element